packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -554,18 +561,28 @@ export interface GitHubSourceProps extends ThirdPartyGitSourceProps {
    * @example 'aws-cdk'
    */
   readonly repo: string;
+
+  /**
+   * The name of the branch.
```

Comment:
Can you also specify that you can give tags, commit IDs, and PR IDs, according to http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codebuild-project.html#cfn-codebuild-project-sourceversion ?

Reply:
Makes sense. I renamed the property so "sourceVersion" to be more general and updated the doc to reflect the variety of options.